designs/service_function_chaining/SFC_design.md

```diff
@@ -133,32 +156,111 @@ Proposed API Definition
 ```
 ElementIdInfo registerInspectionPort(InspectionPortElement inspectionPort) throws NetworkPortNotFoundException, Exception;
 ```
+###### installInspectionHook()
+This API uses updated `InspectionPortElement` . This interfaces parentId argument carries either NULL in case of no port chain is created at target Openstack nodes or an already existing port chain Id to add more flow classifiers to that port chain ID. And the same parentId is used for `installInspectionHookId`. Detail installInspectionHook work flow is show below. `networkElement` will be a list of elements.  
+
+Current API definition  
+
+```
+String installInspectionHook(NetworkElement networkElem, InspectionPortElement inspectionPort, Long tag,
+            TagEncapsulationType encType, Long order, FailurePolicyType failurePolicyType)
+                    throws NetworkPortNotFoundException, Exception;
+```
+Proposed API Definition  
+
+```
+String installInspectionHook(List NetworkElement networkElem , InspectionPortElement inspectionPort,

```

Comment:
Not able to create exact List <interface name > kind of syntax in mark down . Hence just said List interface name. That  is <...> this notation is giving error or weird look.